magda-web-server/src/index.ts

```diff
@@ -97,7 +103,9 @@ app.use(
                 "platform.twitter.com",
                 "www.googletagmanager.com",
                 "www.google-analytics.com",
-                "rum-static.pingdom.net"
+                "rum-static.pingdom.net",
+                "cdnjs.cloudflare.com", //rollbar and shims
+                "wzrd.in" // es7 shim
```

Comment:
Although admittedly we're allowing unsafe-inline at the moment so it's the wild west, it'd be cool if we could be a bit more specific with these last two seeing how any javascript can be allowed on them. According to [this](https://stackoverflow.com/questions/33680150/does-the-content-security-policy-standard-support-wildcard-paths-if-not-why-do) you can put directories into the CSP rather than just domains - can we do `cdnjs.cloudflare.com/ajax/libs/es5-shim/` etc instead?